frameworks/JavaScript/koa/bash_profile.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+export NODE_HOME=${IROOT}/node-v0.12.0-linux-x64
```

Comment:
We've removed the need for bash_profile.sh from TFB. If you could rebase and move NODE_HOME to setup that would be helpful.